lib/grn_ecmascript.lemon

```diff
@@ -417,9 +417,18 @@ primary_expression ::= STRING.
 primary_expression ::= BOOLEAN.
 primary_expression ::= NULL.
 
+
 array_literal ::= BRACKETL elision BRACKETR.
 array_literal ::= BRACKETL element_list elision BRACKETR.
 array_literal ::= BRACKETL element_list BRACKETR.
+array_literal ::= BRACKETL BRACKETR. {
+  grn_ctx *ctx = efsi->ctx;
+
+  GRN_PTR_INIT(&efsi->array_literal, GRN_OBJ_VECTOR, GRN_ID_NIL);
+  grn_expr_take_obj(ctx, efsi->e, &efsi->array_literal);
```

Comment:
We should not pass static object to `grn_expr_take_obj()`. We should pass heap allocated object created by `grn_obj_open()`.